pkg/agent/openflow/packetin.go

```diff
@@ -78,9 +78,11 @@ const (
 
 	// We use OpenFlow Meter for packetIn rate limiting on OVS side.
 	// Meter Entry ID.
-	PacketInMeterIDNP  = 1
-	PacketInMeterIDTF  = 2
-	PacketInMeterIDDNS = 3
+	// 1-255 are reserved for Egress QoS. The Egress QoS meterID leverage the same
```

Comment:
the comment should mention that Antrea limits the number of Egress IPs pers Node to 255, hence the reserved range

Reply:
Done.